src/value.h

```diff
@@ -260,6 +260,9 @@ static inline int RSValue_BoolTest(RSValue *v) {
     case RSValue_String:
       return v->strval.len != 0;
     case RSValue_RedisString: {
+      if (v->rstrval == NULL) {
```

Comment:
So we are not going to cherry pick the is_null?

Reply:
@MeirShpilraien will do. waiting to finalize on master

Reply:
So we do not really need this line then right?

Reply:
@MeirShpilraien 
it failed without it
Will add isnull here as well

Reply:
In 1.4, `evalProperty` always return `RSValue_RedisString`. If field does not exist, it contains `null`